.gitlab-ci.yml

```diff
@@ -922,6 +939,41 @@ iot_agent_deb-arm64:
     paths:
       - $OMNIBUS_PACKAGE_DIR
 
+iot_agent_deb-armhf:
+  stage: package_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_armhf:$DATADOG_AGENT_ARMBUILDIMAGES
+  tags: ["runner:docker-arm", "platform:arm64"]
+  needs: [ "fail_on_non_triggered_tag", "build_iot_agent-deb_armhf"]
+  variables:
+    AGENT_MAJOR_VERSION: 7
+    PYTHON_RUNTIMES: '3'
```

Comment:
What do we need these variables for? I don't see them used in the body of the script nor inside `skip_when_unwanted_on_7`

Reply:
I think it was simply copy-pasted from the job above, but indeed this job doesn't need `AGENT_MAJOR_VERSION` (`--major-version` is set to 7 below) nor `PYTHON_RUNTIMES` (the iot agent doesn't ship the python runtimes). I think the `arm64` IoT Agent jobs also do not need these variables.